### PR TITLE
Show toast on upload error

### PR DESCRIPTION
**Pull request by ds300**
A little toast for when image uploads fail. Solves #2944

Kapture 2024-02-27 at 09 27 12

### Change Type

- [x] `patch` — Bug fix

### Release Notes

- Adds a quick toast to show when image uploads fail.

### CODE REVIEW

File: assets/translations/main.json [hunk 1]
```diff
@@ -101,6 +101,8 @@
 	"action.zoom-to-100": "Zoom to 100%",
 	"action.zoom-to-fit": "Zoom to fit",
 	"action.zoom-to-selection": "Zoom to selection",
+	"assets.files.upload-failed": "Upload failed",
+	"assets.url.failed": "Extracting bookmark info failed",
```
Comment (SomeHats) [inline assets/translations/main.json:105]: This feels a bit technical - what about "Couldn't load URL preview" or "Getting preview failed" or something like that?

**Review by SomeHats — APPROVED** [review]
love it!

**Comment by mimecuvalo** [comment]
can i suggest a tweak here @ds300? feels like this should have some semantics around severity: success, error, warning, etc. https://mui.com/material-ui/react-alert/#severity

i'm happy to riff on this myself or lemme know if you wanna take that on

**Comment by ds300** [comment]
@mimecuvalo not planning to revisit this myself so you go ahead.

**Comment by steveruizok** [comment]
- Introduce severity property (it's optional)
- Severity property sets the color of the icon and default icon
- User provided icon takes precedence (ie other icon or null)
- No severity = no icon / color

**Comment by steveruizok** [comment]
From radix icons, use:
- check-circle
- info-circle
- triangle-exclamation

**Pull request merged**